rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
You need to add the rule to a category (done in the category file)

Reply:
> You need to add the rule to a category (done in the category file)

Done - added the rule to software-development category

Reply:
I think it should be in rules to better GitHub as well

Reply:
@wicksipedia 

> I think it should be in rules to better GitHub as well

Done - added to rules to better github